FEATURE: copy fusion path from breadcrumb

PRGfx opened the pull request:
I like the option to copy the path from the yaml Settings from the Breadcrumb. This PR adds the possibility to copy the fusion path from the breadcrumb in fusion files. I miss this feature when copying a path to override.

Some things I dislike about the current breadcrumb situation:
- I pretty much duplicated the label generation for a breadcrumb item, because the `getElementInfo` does not quite match my needs. Maybe it would make sense to generate the label with some options like add `prototype(%s)` formatting. However this seems oddly specific to me.
- I think the breadcrumb should show "normal" paths as well to allow copying something like `….@process.key`, however I haven't yet understood how the `FusionElement`s relate to each other and maybe a lot falls apart with things like afx blocks?
- Maybe `getElementTooltip` should display the "full" path part name (as generated as per the first point) iff it would be truncated by `getElementInfo`? I didn't want to touch anything there yet, in case there is more reasoning behind the current implementation.

Maybe I can add a test as well, but first I wanted to see whether there are things on this topic that I haven't taken into account yet.

cvette commented:
I've added path parts to the breadcrumb. That was a bit tricky, since the grammar needed to be adjusted. 
Also added some icons, like in the structure view.

PRGfx commented:
> I've added path parts to the breadcrumb. That was a bit tricky, since the grammar needed to be adjusted. Also added some icons, like in the structure view.

I was afraid this thing would have further implications..

I have tested it and it really looks like a step up for the fusion breadcrumb 👍  Thank you for your work.

To be honest I have mixed feelings about the icons though.. I like how you made it consistent with the structure tree, but I think the path icons add little value for the space they're taking up.
As there is no further information displayed or even available (most path parts are just a prototype instantiation -- sometimes we have this information, sometimes it is inherited or implied (as in nested `DataStructure`s or in `Case`)), I feel like almost everything after the first prototype would usually be just a "normal" path segment.
I see how meta and prototype (should) stand out (especially prototype, as the breadcrumb doesn't include the `prototype(...)`), but on the other hand I see how it looks a bit off if there are some items with icons and some without:
![grafik](https://user-images.githubusercontent.com/3868790/184665258-d9a22b5f-1b74-4c34-8705-ebadc3d4a161.png)

cvette commented:
It's fine by me if we remove the icons for normal paths. Doesn't look too bad
